src/components/v2/appDetails/k8Resource/nodeType/Node.component.tsx

```diff
@@ -202,66 +202,70 @@ function NodeComponent({ handleFocusTabs, externalLinks, monitoringTools, isDevt
-                    >
-                        <div onClick={(e) => stopPropagation(e)}>
+                    <>
+                        <div>
                             <span>
-                                {node.name}.{node.namespace}:
-                            </span>
-                            <span className="fs-13 dc__truncate-text mw-18 cursor">
-                            {node.port[0]}  
-                                <span>
-                                    <Clipboard
-                                        className="resource-action-tabs__clipboard icon-dim-12 pointer ml-4 mr-4"
-                                        onClick={(event) => {
-                                            toggleClipBoard(event, node.name)
-                                        }}
-                                    />
-                                </span>
-                                +{node.port.length - 1} more
+                                {node.name}.{node.namespace}:{node.port[0]}
                             </span>
                         </div>
-                    </TippyCustomized>
+                        <span>
+                            <Clipboard
+                                className="resource-action-tabs__clipboard icon-dim-12 pointer ml-8 mr-8"
+                                onClick={(event) => {
+                                    toggleClipBoardPort(event, node.name.concat(":", node.port))
+                                }}
+                            />
+                        </span>
+                        <TippyCustomized
+                            hideHeading={true}
+                            noHeadingBorder={true}
+                            theme={TippyTheme.white}
+                            className="default-tt p-12"
+                            arrow={false}
+                            placement="bottom"
+                            trigger="click"
+                            additionalContent={additionalTippyContent(node)}
+                            interactive={true}
+                        >
+                            <div>
+                                <span className="dc__link dc__link_over dc__ellipsis-right cursor" data-key={node.name}>
+                                    +{node.port.length - 1} more
+                                </span>
+                            </div>
+                        </TippyCustomized>
+                    </>
                 )
             } else if(node.port?.length ===  1){
-                return `${node.name}.${node.namespace} : ${node.port}`
+                return `${node.name}.${node.namespace}:${node.port}`
             } else {
                 return "Port Number is missing"
             }
         }
 
         let _currentNodeHeader = ''
-        const renderClipboardInteraction = (nodeName: string): JSX.Element => {
+        const renderClipboardInteraction = (nodeName: string, node?:any): JSX.Element => {
```

Comment:
This is conditional so that we can use for the conditional visibility of copy icon <Clipboard, removed this since this of no use here. Figured out an alternative way to conditionally render that.